frontend/javascripts/messages.ts

```diff
@@ -53,9 +53,9 @@ export const settingsTooltips: Partial<Record<keyof RecommendedConfiguration, st
     "Decrease size of transferred data by half, using lossy compression. Recommended for poor and/or capped internet connections.",
   interpolation: "Smooth the rendered data by interpolating color values",
   renderMissingDataBlack:
-    "If disabled, missing data will be rendered by using downsampled resolutions.",
+    "If disabled, missing data will be rendered by using downsampled magnifications.",
   gpuMemoryFactor:
-    "Controls which data resolution is displayed, depending on zoom step and viewport size. Adapt this setting to your hardware, so that rendering quality and performance are balanced. Medium is the default. Choosing a higher setting can result in poor performance.",
+    "Controls which data magnification is displayed, depending on zoom step and viewport size. Adapt this setting to your hardware, so that rendering quality and performance are balanced. Medium is the default. Choosing a higher setting can result in poor performance.",
```

Comment:
```suggestion
    "Controls which data magnification is displayed, depending on zoom value and viewport size. Adapt this setting to your hardware, so that rendering quality and performance are balanced. Medium is the default. Choosing a higher setting can result in poor performance.",
```